### PR TITLE
release/4.13.0

### PR DESCRIPTION
**Pull request by luu-alex**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](https://github.com/web3/web3.js/blob/4.x/.github/CONTRIBUTING.md).

#### web3-core

-   Adds a new property (`customTransactionSchema`) to `Web3ConfigOptions` (#7227)

#### web3-eth

-   Adds the same `{transactionSchema?: ValidationSchemaInput}` that exists in `formatTransaction` to `validateTransactionForSigning`

#### web3-eth

-   Forwards the new `web3Context.config.customTransactionSchema` to `formatTransaction`

#### web3-eth-personal

-   Forwards the new `web3Context.config.customTransactionSchema` to `formatTransaction`

#### web3

-   `Web3.providers` namespace exports `type EIP6963ProviderResponse = Map<string, EIP6963ProviderDetail>`. Return type for the static `Web3.requestEIP6963Providers` is now `Promise<EIP6963ProviderResponse>`. (#7239)
-   The callback function provided to the static `Web3.onNewProviderDiscovered` function expects a parameter of type `EIP6963ProvidersMapUpdateEvent` as opposed to `EIP6963AnnounceProviderEvent`. (#7242)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

**Review by avkos — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2701,14 +2701,14 @@ If there are any bugs, improvements, optimizations or any new feature proposal f
...
+## [4.13.0]
 
 ### Added
 
 #### web3-core
 
--   Adds a new property (`customTransactionSchema`) to `Web3ConfigOptions`
--   Adds a new property (`config`) to `Web3RequestManager`
+-   Adds a new property (`customTransactionSchema`) to `Web3ConfigOptions` (#7227)
+-   Adds a new property (`config`) to `Web3RequestManager` (#7227)
```
Comment (nicolasbrugneaux) [inline CHANGELOG.md]: Seems I forgot to remove that line for the change log in 
- #7227 
```suggestion
```

File: packages/web3-core/CHANGELOG.md [hunk 2]
```diff
@@ -233,9 +233,11 @@ Documentation:
...
 
-## [Unreleased]
+## [4.6.0]
 
 ### Added
 
--   Adds a new property (`customTransactionSchema`) to `Web3ConfigOptions`
--   Adds a new property (`config`) to `Web3RequestManager`
+-   Adds a new property (`customTransactionSchema`) to `Web3ConfigOptions`(#7227)
+-   Adds a new property (`config`) to `Web3RequestManager` (#7227)
```
Comment (nicolasbrugneaux) [inline packages/web3-core/CHANGELOG.md]:
```suggestion
```


Same as above

Comment (jdevcs) [reply packages/web3-core/CHANGELOG.md]: @luu-alex

File: packages/web3-eth-personal/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "web3-eth-personal",
-	"version": "4.0.8",
+	"version": "4.0.9",
```
Comment (jdevcs) [inline packages/web3-eth-personal/package.json]:
```suggestion
	"version": "4.1.0",
```

**Comment by jdevcs** [comment]
@luu-alex pls also include web3-types pkg in this release as it was not in last hot fix

File: packages/web3-eth-personal/package.json (hunk 3 above)
Comment (luu-alex) [reply packages/web3-eth-personal/package.json]: updated

**Review by jdevcs — APPROVED** [review]

**Pull request merged**